sci-libs/cfitsio/cfitsio-3.47.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A FITS File Subroutine Library"
+DESCRIPTION="CFITSIO is a library of ANSI C routines for reading and writing
+FITS format data files.  A set of Fortran-callable wrapper routines are also
+included for the convenience of Fortran programmers."
+HOMEPAGE="https://heasarc.gsfc.nasa.gov/docs/software/fitsio/fitsio.html"
+COPYRIGHT="1996-2019 William D. Pence - HEASARC, NASA/GSFC"
+LICENSE="CFITSIO"
+REVISION="1"
+SOURCE_URI="http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-$portVersion.tar.gz"
+CHECKSUM_SHA256="418516f10ee1e0f1b520926eeca6b77ce639bed88804c7c545e74f26b3edf4ef"
+PATCHES="cfitsio-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cfitsio$secondaryArchSuffix = $portVersion
+	lib:libcfitsio$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	cfitsio${secondaryArchSuffix}_devel = $portVersion
+	devel:libcfitsio = $portVersion
+	"
+REQUIRES_devel="
+	cfitsio$secondaryArchSuffix == $portVersion
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libgfortran$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:make
+	"
+
+TEST_REQUIRES="
+	cmd:diff
+	"
+
+BUILD()
+{
+	if [ $effectiveTargetArchitecture == x86_64 ]; then
+		GCC5FLAGS="--enable-x86-64"
+	fi
+
+	autoconf
+	runConfigure ./configure --enable-reentrant --enable-ssse3 --with-bzip2
```

Comment:
Target Northwood on x86 and Prescott on 64 bit.
Ie.: Northwood got no SSE3

Reply:
done!